Extensions/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.nuspec

```diff
@@ -15,5 +15,6 @@
     <file src="../../../LICENSE.txt" />
     <file src="Artifacts/**" target="."/>
     <file src="bin\$configuration$\netcoreapp3.1\EdFi.Ods.Extensions.Homograph.dll" target="."/>
+	<file src="assemblyMetadata.json" target="."/>
```

Comment:
As per the acceptance criteria on the [JIRA ticket](https://tracker.ed-fi.org/browse/ODS-4664), the _assemblyMetadata.json_ file is supposed to be added as an embedded resource in the assembly. As such, this PR just needs to be closed.

As a side note, make sure you are adding white space using spaces. There's a <kbd>TAB</kbd> in each of the changes on this PR.